frontend/javascripts/oxalis/api/api_latest.js

```diff
@@ -1040,8 +1043,26 @@ class DataApi {
    * @example
    * const segmentationOpacity = api.data.getConfiguration("segmentationOpacity");
    */
-  getConfiguration(key: $Keys<DatasetConfiguration>) {
-    return Store.getState().datasetConfiguration[key];
+  getConfiguration(key: $Keys<DatasetConfiguration> | OutdatedDatasetConfigurationKeys) {
+    switch (key) {
```

Comment:
I think I wouldn't do it right now, because the change is relatively minor and from what I just checked there are still api scripts in very active use that request API version 2. Bumping the latest version now would make those fail. We should push people (again) to use the newest API version before doing that out of the blue.
I think the specific deprecation warnings are a good thing, so I would include them in any case. Script writers will see more easily what they need to migrate :)